hamilton/plugins/h_ray.py

```diff
@@ -209,20 +209,33 @@ class RayTaskExecutor(executors.TaskExecutor):
     This is still experimental, so the API might change.
     """
 
-    def __init__(self, num_cpus: int):
+    def __init__(
+        self,
+        num_cpus: int = None,
```

Comment:
We expose num_cpus -- anything else we want to expose? Or do we just want to keep that there for backwards compatibility?

Reply:
backwards compatibility.